package.json

```diff
@@ -104,7 +106,13 @@
   },
   "eik": {
     "server": "https://assets.finn.no",
-    "files": "dist"
+    "type": "package",
+    "files": "eik",
+    "import-map": [
+      "https://assets.finn.no/map/react/v2",
+      "https://assets.finn.no/map/vue/v1",
+      "https://assets.finn.no/map/custom-elements/v2"
```

Comment:
Are we sure about this url path? Where does it come from? I'm mostly curious about the `v2` part of it.

Reply:
Yes, it's from here:
https://github.schibsted.io/nmp-web/eik-import-maps/blob/main/maps/custom-elements/eik.json

Not sure why they moved to v2, but that's the latest :)

Reply:
Those paths are the same as the one I found in @fabric-ds/icons: https://github.com/fabric-ds/icons/blob/main/package.json#L69-L72
It doesn't sound correct to override anything on those paths 🤔

Reply:
Ah, nevermind that, that's the version of maps/custom-elements, which hold import maps for different packages, including @fabric-ds and @warp-ds. https://github.schibsted.io/nmp-web/eik-import-maps/blob/main/maps/custom-elements/map.json